listenbrainz/db/year_in_music.py

```diff
@@ -298,31 +300,24 @@ def notify_yim_users(year):
         year_in_music = f"{base_url}/user/{user_name}/year-in-music/"
         params = {
             "user_name": user_name,
-            "year_in_music": year_in_music,
-            "statistics": f"{base_url}/user/{user_name}/reports/",
-            "feed": f"{base_url}/feed/",
-            "feedback": f"{base_url}/user/{user_name}/feedback/",
-            "pins": f"{base_url}/user/{user_name}/pins/",
-            "playlists": f"{base_url}/user/{user_name}/playlists/",
-            "collaborations": f"{base_url}/user/{user_name}/collaborations/",
-            "lb_logo_cid": lb_logo_cid[1:-1]
+            "logo_cid": logo_cid[1:-1]
         }
 
         try:
             send_mail(
-                subject="Year In Music 2021",
+                subject=f"Year In Music {year}",
                 text=render_template("emails/year_in_music.txt", **params),
                 to_email=row["email"],
                 to_name=user_name,
                 html=render_template("emails/year_in_music.html", **params),
-                lb_logo_cid=lb_logo_cid,
-                lb_logo=lb_logo
+                logo_cid=logo_cid,
+                logo=logo
             )
         except Exception:
             current_app.logger.error("Could not send YIM email to %s", user_name, exc_info=True)
 
         # create timeline event too
-        timeline_message = 'ListenBrainz\' very own retrospective on 2021 has just dropped: Check out ' \
+        timeline_message = f'ListenBrainz\' very own retrospective on {year} has just dropped: Check out ' \
```

Comment:
Shouldn't it be `ListenBrainz\'s`? Or is the idea that the z is pronounced as an s?

Reply:
Yes, that is how I've always used it. A little bit playful if you ask me.